lib/grape/formatters/active_model_serializers.rb

```diff
@@ -0,0 +1,18 @@
+# A grape response formatter that can be used as 'formatter :json, Grape::Formatters::ActiveModelSerializers'
+#
+# Serializer options can be passed as a hash from your grape endpoint using env[:active_model_serializer_options],
+# or better yet user the render helper in Grape::Helpers::ActiveModelSerializers
+module Grape
+  module Formatters
+    module ActiveModelSerializers
+      class << self
+        def call(resource, env)
```

Comment:
Look great.  

1. would you mind changing this to `def self.call(resource, env)`?  I don't think there's any advantage to opening the singleton class here
1. could you add the documentation from `lib/grape/active_model_serializers.rb` to the docs or README as you see fit.  (I think there's an integrations page in progress somewhere)
1. Could you add a test so we can better support this code?